src/games/strategy/triplea/ai/proAI/util/ProSortMoveOptionsUtils.java

```diff
@@ -54,7 +54,8 @@ public int compare(final Map.Entry<Unit, Set<Territory>> o1, final Map.Entry<Uni
   }
 
   public static Map<Unit, Set<Territory>> sortUnitNeededOptions(final PlayerID player,
-      final Map<Unit, Set<Territory>> unitAttackOptions, final Map<Territory, ProTerritory> attackMap) {
+      final Map<Unit, Set<Territory>> unitAttackOptions, final Map<Territory, ProTerritory> attackMap,
+      final ProOddsCalculator calc) {
     final GameData data = ProData.getData();
 
     final List<Map.Entry<Unit, Set<Territory>>> list =
```

Comment:
In this case, I didn't touch that code so definitely out of scope of this PR. You could open an issue if you think it is an actual bug. Though most likely it is just something that could be optimized. A good portion of the sort code actually could use some optimization but I wouldn't say any of it is a bug or high priority.